src/core/RandomVariables.jl

```diff
@@ -946,13 +968,35 @@ end
                     end
                 else
                     @warn("[Turing] attempt to invlink an invlinked vi")
-                end    
+                end
             end)
         end
     end
     return expr
 end
 
+
+"""
+    islinked(vi::VT, spl::Sampler) where VT<:VarInfo
+
+Returns `true` if a `VarInfo` is linked for a particular sampler `spl`.
```

Comment:
Updated to say "in the transformed space" rather than "linked", is that helpful?

Reply:
It's helpful, but maybe add a reference to some other documentation for Bijectors or something? I mean, it's clear what's happening here if you know anything about the relationship between Turing and Bijectors etc, but for a newbie it might still be a bit unclear?

Reply:
Updated to read:

    islinked(vi::VT, spl::Sampler) where VT<:VarInfo

Returns `true` if a `VarInfo` is in the transformed space for a particular sampler `spl`.

Turing's Hamiltonian samplers use the `link` and `invlink` functions from 
[Bijectors.jl](https://github.com/TuringLang/Bijectors.jl) to map a constrained variable
(for example, one bounded to the space `[0, 1]`) from its constrained space to the set of 
real numbers. `islinked` checks if the number is in the constrained space or the real space.